src/EnergyPlus/HVACHXAssistedCoolingCoil.cc

```diff
@@ -1346,11 +1153,8 @@ namespace HVACHXAssistedCoolingCoil {
         // incorrect coil type or name is given, ErrorsFound is returned as true and capacity is returned
         // as negative.
 
-        // Using/Aliasing
-        auto &GetDXCoilCapacity(DXCoils::GetCoilCapacity);
```

Comment:
[src/EnergyPlus/HVACHXAssistedCoolingCoil.cc:1350]:(style),[constVariable],Variable 'GetDXCoilCapacity' can be declared as reference to const